sdk/tables/Azure.Data.Tables/src/TableServiceClient.cs

```diff
@@ -273,10 +273,12 @@ public TableServiceClient(Uri endpoint, TokenCredential tokenCredential, TableCl
             var perCallPolicies = _isCosmosEndpoint ? new[] { new CosmosPatchTransformPolicy() } : Array.Empty<HttpPipelinePolicy>();
             var endpointString = _endpoint.AbsoluteUri;
             string secondaryEndpoint = TableConnectionString.GetSecondaryUriFromPrimary(_endpoint)?.AbsoluteUri;
-
+            var audienceScope = string.IsNullOrEmpty(options?.Audience?.ToString())
+                ? TableAudience.AzurePublicCloud.GetDefaultScope(_isCosmosEndpoint)
+                : options.Audience.Value.GetDefaultScope(_isCosmosEndpoint);
```

Comment:
```suggestion
            var audienceScope = (options?.Audience ?? TableAudience.AzurePublicCloud)
                .GetDefaultScope(_isCosmosEndpoint);
```

I think we can simplify a bit.  We know that the default is `null`.  If someone creates an explicit instance with an empty string as the value, there's no need to protect them and fallback.   Let them fail.